src/chart/generateCategoricalChart.tsx

```diff
@@ -2040,6 +2041,24 @@ export const generateCategoricalChart = ({
       );
     };
 
+    static renderActiveBar = (option: any, props: any): React.ReactElement => {
```

Comment:
So I know the other active components and a lot of others have many uses of `any` - is there any probable way that we are able to type this correctly?

Trying to enforce not using `any` where possible

Reply:
I agree with you. This was a good ask. Take another look. I think I have something with a little more type safety?